patches/server/0010-Adventure.patch

```diff
@@ -4895,6 +4895,21 @@ index 5540a766bc7edf8985058e29cc419c2c6b1d4428..baff11728120fda1740d5e15163760f5
  
      @Override
      public String getDisplayName() {
+@@ -303,4 +360,14 @@ final class CraftTeam extends CraftScoreboardComponent implements Team {
+         return !(this.team != other.team && (this.team == null || !this.team.equals(other.team)));
+     }
+ 
++    // Paper - make Team extend ForwardingAudience
++    @Override
++    public @org.jetbrains.annotations.NotNull Iterable<? extends net.kyori.adventure.audience.Audience> audiences() {
++        java.util.List<net.kyori.adventure.audience.Audience> audiences = new java.util.ArrayList<>();
++        for (String playerName : this.team.getPlayers()) {
++            audiences.add(Bukkit.getPlayerExact(playerName));
```

Comment:
It is very not smart to add null into that iterable.
ForwardingAudience#sendMessage for-loops over that and would promptly NPE.

Reply:
Should I do `getOfflinePlayer` instead? That would match `CraftTeam#getPlayers()`

Reply:
Does it extend Audience?

Reply:
> Does it extend Audience?

nope lol i didn't say anything :p

Reply:
I cannot reproduce this though. 
I just used a modified version of the getPlayers defined in CraftTeam:
```java
@Override
public @NotNull Iterable<? extends Audience> audiences() {
    this.checkState();

    ImmutableSet.Builder<org.bukkit.entity.Player> players = ImmutableSet.builder();
    for (String playerName : this.team.getPlayers()) {
        org.bukkit.entity.Player player = Bukkit.getPlayerExact(playerName);
        if (player != null) {
            players.add(player);
        }
    }
    return players.build();
}
```